### DISCUSSION TITLE
Problemas a dubugar no VS

### DISCUSSION THREAD

**Discussion by ragnarside** [Flutterando/forum#80] [Perguntas e Respostas]
Boa noite comunidade,

estou iniciando meus estudos no FLUTTER e ao tentar debugar pelo vs deu um erro no console de depuração, se alguém puder me ajudar eu agradeço. Obrigado.

Launching lib\main.dart on Android SDK built for x86 in debug mode...
lib\main.dart:1
ziparchive W 11-14 21:01:09 22420 14220] Unable to open 'C:\Users\jascd\OneDrive\Área de Trabalho\exemplo\build\app\outputs\flutter-apk\app.apk': Illegal byte sequence
zipro W 11-14 21:01:09 22420 14220] Error opening archive C:\Users\jascd\OneDrive\Área de Trabalho\exemplo\build\app\outputs\flutter-apk\app.apk: I/O error
ERROR: dump failed because resource AndroidManifest.xml not found
Failed to extract manifest from APK: ProcessException: The command failed
  Command: C:\Users\jascd\AppData\Local\Android\sdk\build-tools\33.0.0\aapt dump xmltree C:\Users\jascd\OneDrive\Área de Trabalho\exemplo\build\app\outputs\flutter-apk\app.apk AndroidManifest.xml.
No application found for TargetPlatform.android_x86.
Is your project missing an android\app\src\main\AndroidManifest.xml?
Consider running "flutter create ." to create one.
Exited (sigterm)

**Comment by DouglasValerio** [comment]
Eu imagino que tu checou e esse arquivo do `AndroiManifest.xml` está lá no `android\app\src\main\AndroidManifest.xml` certo?

Nesse caso, como vi que parece que tua pasta faz sync com o OnDrive, pode ser problema de permissão de acesso, no qual os comandos de build e run do flutter não estão com autorização de leitura/escrita nessa sua pasta do Onedrive.

Eu tentaria mover o projeto pra uma pasta que não sincronize com o OneDrive. Muito tempo atrás, tive problemas derivados disso, por alguma razão o sync removia a autorização de acesso ao arquivo mesmo ele existindo, assim o flutter diz que não existe mesmo ele existindo.

**Reply by ragnarside** [reply]
Sim, eu verifiquei e o xml está lá. Sobre o One drive, eu não tinha me atentado a isso, porém fiz a desinstalação dele pra não me atrapalhar mais, por que realmente vi no curso que isso poderia afetar bastante coisa mais pra frente, porém mesmo depois de realizar esses procedimentos, ainda continua dando um erro. Mas agora da esse:

Launching lib\main.dart on Windows in debug mode...
lib\main.dart:1
Exception: Build process failed.
Exited (sigterm)

**Reply by DouglasValerio** [reply]
Tu tá tentando rodar o app pro Windows? Esse erro parece indicar que ele tá tentando buildar pro Windows e falhando
roda um `flutter doctor -v` e posta aqui.
Em teoria, se o emulador do Android estiver rodando, o `flutter run` vai buildar e rodar o app em modo debug direto no emulador, mas pode ser qua alguma config esteja inválida, fazendo com que o emulador não seja encontrado.
